src/main/java/com/esri/samples/symbology/read_symbols_from_mobile_style_file/ReadSymbolsFromMobileStyleFileSample.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2019 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.esri.samples.symbology.read_symbols_from_mobile_style_file;
+
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.stage.Stage;
+
+public class ReadSymbolsFromMobileStyleFileSample extends Application {
+
+  private static ReadSymbolsFromMobileStyleFileController controller;
+
+  @Override
+  public void start(Stage stage) throws Exception {
+
+    // set up the scene
+    FXMLLoader loader = new FXMLLoader(getClass().getResource("/fxml/read_symbols_from_mobile_style_file.fxml"));
+    Parent root = loader.load();
+    controller = loader.getController();
+    Scene scene = new Scene(root);
+
+    // set title, size, and add scene to stage
+    stage.setTitle("Read Symbols From Mobile Style File Sample");
+    stage.setWidth(800);
+    stage.setHeight(700);
+    stage.setScene(scene);
+    stage.show();
+  }
+
+  /**
+   * Stops and releases all resources used in application.
+   */
+  @Override
```

Comment:
the `stop` method should always be the last method in the launcher class, for consistency with our other samples.